### DISCUSSION TITLE
Καταγραφή απαντήσεων στα κουιζ

### DISCUSSION THREAD

**Discussion by StavroulaAnatolaki** [courses-ionio/hci#1603] [Announcements]
Κυριε στο EdPuzzle βλεπω πως το βιντεο που μιλησαμε την Παρασκευη μου λεει πως δεν απαντησα στις ερωτησεις του ενω το εκανα. Τι συμβαινει;

**Comment by epidrome** [comment]
δεν μπορούμε να δούμε τι βλέπετε εσείς!

μπορείτε να ενημερώσετε το ερώτημα σας με το αντίστοιχο screenshot;

**Reply by StavroulaAnatolaki** [reply]
δεν καταλαβα τι εννοειτε

**Reply by epidrome** [reply]
θα μας βοηθούσε μια φωτογραφία τι βλέπετε στην οθόνη σας;

**Reply by StavroulaAnatolaki** [reply]
Δυσκολεύομαι να στείλω φωτογραφία

**Reply by StavroulaAnatolaki** [reply]
Απλώς λέει πως το βίντεο που συζητήσαμε αυτή την εβδομάδα λέει unfinished. Ενώ εγώ απάντησα στις ερωτήσεις του πριν τη προθεσμία

**Reply by Aggeloskourtesis** [reply]
Αν στο "Due date"  δεν είναι με κόκκινο η ημερομινία δεν θα έχεις θέμα.

**Reply by p14kairas** [reply]
Μετά την τελευταία ερώτηση πρέπει να συνεχίσεις  και να δεις απλά το υπόλοιπο βίντεο μέχρι να τελειώσει. @StavroulaAnatolaki
